ziti/cmd/ascode/exporter/exporter.go

```diff
@@ -86,52 +163,27 @@ func NewExportCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 
 	v.AutomaticEnv()
 
-	edge.AddLoginFlags(cmd, &exporter.loginOpts)
+	edge.AddLoginFlags(cmd, &loginOpts)
 	cmd.Flags().SetInterspersed(true)
-	cmd.Flags().BoolVar(&exporter.ofJson, "json", true, "Output in JSON")
-	cmd.Flags().BoolVar(&exporter.ofYaml, "yaml", false, "Output in YAML")
-	cmd.Flags().StringVar(&exporter.loginOpts.ControllerUrl, "controller-url", "", "The url of the controller")
-	cmd.MarkFlagsMutuallyExclusive("json", "yaml")
+	cmd.Flags().StringVar(&outputFormat, "output-format", "JSON", "Output data as either JSON or YAML (default JSON)")
+	cmd.Flags().StringVar(&loginOpts.ControllerUrl, "controller-url", "", "The url of the controller")
 	ziticobra.SetHelpTemplate(cmd)
 
-	cmd.Flags().StringVarP(&exporter.filename, "output-file", "o", "", "Write output to local file")
-
-	exporter.loginOpts.Out = out
-	exporter.loginOpts.Err = errOut
+	cmd.Flags().StringVarP(&outputFile, "output-file", "o", "", "Write output to local file")
 
 	return cmd
 }
 
-func (exporter *Exporter) Execute(input []string) error {
+func (exporter *Exporter) Execute(client *rest_management_api_client.ZitiEdgeManagement, input []string) (map[string]interface{}, error) {
 
 	logLvl := logrus.InfoLevel
-	if exporter.loginOpts.Verbose {
+	if exporter.verbose {
 		logLvl = logrus.DebugLevel
```

Comment:
it doesn't make sense to change the log level here. it should be established already at this point